iOS/Flows/Conversation/ConversationCoordinator.swift

```diff
@@ -14,7 +14,7 @@ import Combine
 
 class ConversationCoordinator: PresentableCoordinator<Void> {
 
-    lazy var conversationVC = ConversationViewController(conversation: self.conversation, delegate: self)
+    lazy var conversationVC = new_ConversationViewController(conversation: self.conversation)
```

Comment:
Presenting the new conversation VC now. The old VC still exists and you can't use it if needed.